homeassistant/components/tplink/switch.py

```diff
@@ -44,7 +44,7 @@ async def async_setup_entry(
 
 
 class TPLinkSwitch(CoordinatedTPLinkEntity, SwitchEntity):
-    """Representation of a feature-based TPLink sensor."""
+    """Representation of a feature-based TPLink switch."""
```

Comment:
Perhaps leave this out to keep it sensor-only?

Reply:
merged